03-first-aws-environment/README.md

```diff
@@ -0,0 +1,3 @@
+# Tutorial #3: You first AWS Environment with Atmos
+
+The code in this directory is an accompaniment to the [TODO](TODO) tutorial in our SweetOps documentation.
```

Comment:
what will this link to?

Reply:
I'll update with the link to https://github.com/cloudposse/docs/pull/537 when I do updates following our conversations below.